src/packages/excalidraw/CHANGELOG.md

```diff
@@ -14,14 +14,27 @@ Please add the latest change on the top under the correct section.
 
 ## 0.2.0
 
+## Excalidraw API
+
 ### Features
 
-- Add toast [#2772](https://github.com/excalidraw/excalidraw/pull/2772)
-- Add `cmd+o` shortcut to load scene [#2732](https://github.com/excalidraw/excalidraw/pull/2732)
+- Exported few [Extra API's](https://github.com/excalidraw/excalidraw/blob/master/src/packages/excalidraw/README.md#extra-apis) which can be used by the host to communicate with Excalidraw.
+
 - Remove language picker, and add `langCode`, `renderFooter` [#2644](https://github.com/excalidraw/excalidraw/pull/2644):
   - BREAKING: removed the language picker from UI. It is now the host app's responsibility to implement a language picker if desirable, using the newly added [`renderFooter`](https://github.com/excalidraw/excalidraw/blob/master/src/packages/excalidraw/README.md#renderFooter) prop. The reasoning is that the i18n should be controlled by the app itself, not by the nested Excalidraw component.
   - Added [`langCode`](https://github.com/excalidraw/excalidraw/blob/master/src/packages/excalidraw/README.md#langCode) prop to control the UI language.
 - Add support for `exportToBackend` prop to allow host apps to implement shareable links [#2612](https://github.com/excalidraw/excalidraw/pull/2612/files)
+
+### Fixes
+
+- Fix typo for initialData and point all links to master [#2707](https://github.com/excalidraw/excalidraw/pull/2707)
+
+## Excalidraw Library
```

Comment:
What's that?

Reply:
Trying to separate the changelog for Excalidraw API and core library.
Any better suggestion for the title?

Reply:
I thought it was the http://libraries.excalidraw.com/

Reply:
Yea, I would call it something else. Excalidraw App maybe?

Reply:
do you think it can confuse people with the actual excalidraw app? ( i mean we have collab etc separated for excal app). Actually, it shouldn't coz those changes will never be part of this changelog so should be fine then.

Reply:
yea, ok, let's leave it